modules/git/repo_tag.go

```diff
@@ -202,6 +202,13 @@ func parseTagRef(ref map[string]string) (tag *Tag, err error) {
 		}
 	}
 
+	tag.Message = strings.TrimSpace(tag.Message)
+	if tag.Message == "" {
+		tag.Message = tag.Name
+	} else if tag.Name != "" {
+		tag.Message = tag.Name + "\n\n" + tag.Message
+	}
```

Comment:
I think this joining is incorrect, we should not alter the tag message. What was wrong was just the retrieval of the tagged commit message.